src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java

```diff
@@ -0,0 +1,74 @@
+package edu.greenblitz.tobyDetermined.commands.swerve;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class MoveToPos extends SwerveCommand {
+    private ProfiledPIDController xController;
+    private ProfiledPIDController yController;
+    private ProfiledPIDController rotationController;
+    private Pose2d pos;
+
+    public static final double rotKp = 2;
+    public static final double rotKi = 0;
+    public static final double rotKd = 0;
+
+    public static final double translationKp = 2;
+    public static final double translationKi = 0;
+    public static final double translationKd = 0;
+    
+    private double tolerance = 0.05;
+    private double rotationTolerance = 4;
+
+    public MoveToPos(Pose2d pos) {
```

Comment:
Best Practice to add command summary (Get to a location with 3 pid controllers...)